src/Uno.UI/UI/Xaml/Controls/TextBlock/TextBlock.skia.cs

```diff
@@ -47,6 +47,8 @@ public TextBlock()
 			LostFocus += (_, _) => UpdateSelectionRendering();
 		}
 
+		internal bool IsTextBoxDisplay { get; set; }
```

Comment:
Would there be any benefit in marking this property setter as `init` as I see we only initialize it once?

```suggestion
		internal bool IsTextBoxDisplay { get; init; }
```